src/api/demo.rkt

```diff
@@ -60,16 +60,25 @@
                   [("check-status" (string-arg)) check-status]
                   [("timeline" (string-arg)) get-timeline]
                   [("up") check-up]
+                  [("api" "results" (string-arg)) get-results]
                   [("api" "sample") #:method "post" sample-endpoint]
+                  [("api" "sample-start") #:method "post" sample-start-endpoint]
```

Comment:
I mean, these are all fine, it's not the end of the world or anything, but I think it would be a lot nicer to do `/api/start/foo` instead of `/api/foo-start`. We already have a separator character!